rules/game_process.tex

```diff
@@ -215,8 +248,6 @@ \subsection{Kick-off}
 The head referee must signal this from the T-junction of the halfway line.
 \qty{\PlayingDelayTime}{\second} after the head referee has signaled the kick-off, the robots' state is switched to \texttt{playing} by the GameController (regardless of when the ball comes into play).
 
-A goal may be scored directly from a kick-off.
```

Comment:
A goal may be scored directly from a kick-off for the Challenge Shield. However, for the Champions Cup the indirect kick rule applies and the goal does \textit{not} count.